core/cap-0033.md

```diff
@@ -275,11 +290,12 @@ enum UpdateSponsorshipResultCode
     UPDATE_SPONSORSHIP_SUCCESS = 0,
 
     // codes considered as "failure" for the operation
-    UPDATE_SPONSORSHIP_DOES_NOT_EXIST = -1,
-    UPDATE_SPONSORSHIP_NOT_SPONSOR = -2,
-    UPDATE_SPONSORSHIP_LINE_FULL = -3,
-    UPDATE_SPONSORSHIP_LOW_RESERVE = -4,
-    UPDATE_SPONSORSHIP_ONLY_TRANSFERABLE = -5
+    UPDATE_SPONSORSHIP_DOES_NOT_EXIST = -1,   // specified entry does not exist
+    UPDATE_SPONSORSHIP_NOT_SPONSOR = -2,      // not sponsor of specified entry
+    UPDATE_SPONSORSHIP_LOW_RESERVE = -3,      // owner cannot afford this entry
```

Comment:
The term owner is new. There's no field owner that I can see, can we reference who cannot afford the entry specifically?

Reply:
Clarified.